api/wasm-interpreter/src/lib.rs

```diff
@@ -155,14 +157,34 @@ impl WrappedCompiledComp {
         let component = self.0.create_with_canvas_id(&canvas_id).unwrap();
         component.run().unwrap();
     }
-    /// Creates this compiled component in a canvas.
+    /// Creates this compiled component in a canvas, wrapped in a promise.
     /// The HTML must contains a <canvas> element with the given `canvas_id`
     /// where the result is gonna be rendered.
     /// You need to call `show()` on the returned instance for rendering and
     /// `slint.run_event_loop()` loop to make it interactive.
     #[wasm_bindgen]
-    pub fn create(&self, canvas_id: String) -> Result<WrappedInstance, JsValue> {
-        Ok(WrappedInstance(self.0.create_with_canvas_id(&canvas_id).unwrap()))
+    pub fn create(&self, canvas_id: String) -> Result<InstancePromise, JsValue> {
```

Comment:
This is technically a breaking change, isn't it?  Not sure if the wasm is part of our public API though, so maybe it's ok. (at least probably only us were using it, but i've been using it in slides and stuff, but allways through full release URL IIRC)